test/unit/puppetlabs/services/jruby/jruby_testutils.clj

```diff
@@ -52,15 +54,14 @@
   JRubyPuppet pool size with be included with the config, otherwise no size
   will be specified."
   ([]
-   {:ruby-load-path  ruby-load-path
-    :gem-home        gem-home
-    :master-conf-dir conf-dir})
+   (jruby-core/initialize-config
+     {:jruby-puppet {:gem-home        gem-home
+                     :master-conf-dir conf-dir
+                     :master-var-dir  var-dir}
+      :os-settings  {:ruby-load-path ruby-load-path}}))
```

Comment:
Is it weird that `:os-settings` is now in two places in the overall config? The `jruby-puppet-tk-config` function above this adds it to the top-level config too.

Not a big deal, just thinking out loud